### PR TITLE
chore: adjusted versions for release 2.4.9

### PR DESCRIPTION
**Pull request by jacobdotcosta**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -123,22 +123,22 @@
   </pluginRepositories>
 
   <properties>
-    <amqp-10-starter.version>2.3.3</amqp-10-starter.version>
-    <dekorate.version>1.0.3</dekorate.version>
-    <infinispan-starter.version>2.3.3.Final</infinispan-starter.version>
+    <amqp-10-starter.version>2.3.6</amqp-10-starter.version>
```
Comment (aureamunoz) [inline pom.xml:126]: I don't know from where comes the amqp version, don't see it in the ticket.

**Review by aureamunoz — CHANGES_REQUESTED** [review]
From the jira tickets:
- infinispan.version should be 12.1.7.Final, no?
- hibernate.version should be 5.5.7.Final? 
- hibernate-validator.version should be 7.0.1.Final?

File: pom.xml (hunk 1 above)
Comment (jacobdotcosta) [reply pom.xml:126]: Bumped infinispan, hibernate and hibernate-validator versions.

Comment (jacobdotcosta) [reply pom.xml:126]: AMQP version I got if from 2.3.10.

Comment (aureamunoz) [reply pom.xml:126]: ok, althought I think that AMQP should give us a version to use

**Review by aureamunoz — APPROVED** [review]

**Pull request merged**